muxtools/audio/extractors.py

```diff
@@ -386,14 +397,15 @@ def _conv(self, val: int | None):
         if self.trim_use_ms:
             return abs(val) / 1000
         else:
-            return frame_to_timedelta(abs(val), self.fps).total_seconds()
+            return self.resolved_ts.frame_to_time(abs(val), TimeType.EXACT, 3) / 1000
```

Comment:
I'm not sure why you do a ``/ 1000``.
If you wanna get the time without the result to be round in seconds, you can call [``frame_to_time(abs(val), TimeType.EXACT)``](https://github.com/moi15moi/VideoTimestamps/blob/5fa88d05675664b60835e3a348d8ad6630810ade/video_timestamps/abc_timestamps.py#L126-L134). However, it will return a ``Fraction``, not a float.

Reply:
sox just takes seconds as a float. It's mostly just the same behavior as before.
Dunno if I need or even want a fraction for this

Reply:
You can always convert the Fraction to a float.
Currently, the result you get can be rounded (it is rounded for timescale 1000 (a.k.a mkv), but for other timescale, it will be rounded.